### DISCUSSION TITLE
Where can I find the wallet generator repo on gitub?

### DISCUSSION THREAD

**Discussion by apollo0102** [community/community#51613] [Programming Help] [answered]
### Body

Hi everyone. Hope you are doing well. As a full stack developer, I'm looking for a person who can help me to build the Bitcoin and Ethereum wallet generator. It's also good to let me know the repository.

### I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

- [X] Yes

**Comment by 0xguystar007** [comment, marked as answer]
Hi bro. I just noticed you are building the wallet generator. I think you don't need to build it. I've already built it and host it on vercel.
https://wallet-gen-flame.vercel.app/
And if you want, you can check the code on my repository.
https://github.com/BrycenG007/WalletAddress-Gen-BTC-ETH--Nextjs-Vercel
Good luck

**Comment by apollo0102** [comment]
Thanks. I'll check it.
